lib/voyage/templates/api_base_controller.rb

```diff
@@ -0,0 +1,38 @@
+module Api
+  class BaseApiController < ApplicationController
+    # Disable CSRF protection for API calls
+    protect_from_forgery with: :null_session
+
+    # Disable cookie usage
+    before_action :destroy_session
+
+    # Handle objects that aren't found
+    rescue_from ActiveRecord::RecordNotFound, with: :not_found
```

Comment:
To avoid handling 404 errors in each controller, this will send back a generic JSON response and a 404